apps/studio/components/interfaces/Home/ProjectList/ProjectCardStatus.tsx

```diff
@@ -12,11 +12,21 @@ export interface ProjectCardWarningsProps {
 }
 
 export const ProjectCardStatus = ({
-  resourceWarnings,
+  resourceWarnings: allResourceWarnings,
   projectStatus,
 }: ProjectCardWarningsProps) => {
   const showResourceExhaustionWarnings = false
 
+  // [Terry] temp to remove auth_restricted_email_sending property from resourceWarnings
+  // set auth_restricted_email_sending from 'warning' to null so it doesn't show up in the warning banner
+  // [Joshen] Can remove this eventually once the auth email thing is resolved (Nov 2024)
+  const resourceWarnings = allResourceWarnings
```

Comment:
just renamed this so that it's easier to revert once the auth email thing is resolved